code/src/components/Artists.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Artists = (props) => {
+  return (
+    
+    <div>
+      <h3 className="song-artists">        
+        {props.artistArray.map((artist) =>
+        <a 
+        href={artist.external_urls.spotify}
+        target="_blank"
+        rel="noopener noreferrer"
+        >
+        <span>{artist.name}</span></a>)}
```

Comment:
If several artist names, we are missing a comma. There is a possibility to style this in CSS, it would look something like this: `.artist-name:not(:last-child)::after {
  content: ", ";
  color: #a0a0a0;
}`